features/FxA-83-sign-in-confirmation-email/README.md

```diff
@@ -0,0 +1,169 @@
+# Sign-in confirmation email
+
+## Problem statement
+
+It is possible for attackers
+armed with a list of known email/password pairs
+to maliciously access users’ Firefox Accounts.
+In such cases,
+attackers would have full access
+to the user’s Sync data,
+including passwords, history, bookmarks and
+the ability to force-install malicious add-ons.
+
+## Outcomes
+
+If an attack occurs,
+we would like fewer accounts to be compromised
+without negatively impacting
+the number of signed-in users
+or our overall FxA engagement rate.
+
+## Hypothesis
+
+If we introduce an additional email confirmation step
+for Sync sign-ins,
+fewer accounts will be compromised.
+We will know this to be true if,
+in the event of an attack,
+the number of successful sign-ins remains constant
+and the count of 104 (unverified account) errors
+from protected endpoints
+increases in line with any wider increase in traffic.
```

Comment:
I think we also have a secondary hypothesis - that we can add this protection without significantly impacting the success rate of connecting to sync.  And unlike the main hypothesis, we can measure that without having to wait for something bad to happen.